### PR TITLE
Updated look and feel of website

### PR DESCRIPTION
**Pull request by lostlivio**
The original website needed a facelift.

### CODE REVIEW

**Comment by guineveresaenger** [comment]
@lostlivio - 
for your personal sanity and also github workflow standards, could you close this and open a PR with a feature branch from your GH repo?

**Comment by lostlivio** [comment]
@guineveresaenger I wanted to use the same CSS templates we used for cnct.io...makes it easier to maintain both websites.

File: app/views/App/Tracks.html [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+        <p>
+
+              <input name="group1" type="radio" value={{.}} />
+              <span class="blue-text">{{.}}</span>
+
+        </p>
+      {{end}}
+
+      <a class="btn btn-lg btn-primary blue" onclick="document.workload.submit();return false;">Submit</a>
+  </form>
```
Comment (guineveresaenger) [inline app/views/App/Tracks.html]: Radio buttons need to not toggle
I also believe the `name` field is important for the app to pick up the track tag.

**Comment by guineveresaenger** [comment]
Thank you Livio! Looks so much better now. :)

**Pull request merged**